ArduPlane/quadplane.cpp

```diff
@@ -581,7 +581,9 @@ bool QuadPlane::setup(void)
             // this is a duo-motor tailsitter (vectored thrust if tilt.tilt_mask != 0)
             motors = new AP_MotorsTailsitter(plane.scheduler.get_loop_rate_hz(), rc_speed);
             motors_var_info = AP_MotorsTailsitter::var_info;
-            rotation = ROTATION_PITCH_90;
+            if (tilt.tilt_mask <= 0) {
```

Comment:
Won't this break support for tilt-vectored tailsitters?

Reply:
they don't us the tilt mask, I think tailsitters just allays output the left and right tilts

Reply:
that's right, hardwired for a twin engine configuration only. I've avoided thinking about supporting tilt in MatrixMotorsTS :)